CHANGES.md

```diff
@@ -29,6 +29,7 @@
     - Fix `markdown-live-preview-mode` fails when `eww-auto-rename-buffer` is non-nil[GH-737][]
     - Fix to mistake to handle the line as delimiter row[GH-747][]
     - Fix wrong displaying horizontal rule in `markdown-view-mode` [GH-747][]
+    - HTML-escape title in `markdown-add-xhtml-header-and-footer` [GH-748][]
```

Comment:
This should be GH-754 to work properly.

Reply:
Fixed https://github.com/jrblevin/markdown-mode/pull/760/commits/601863bb56a2171b2a4779093ee23bdecdb7dffc